src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+import duke.Exception.DukeException;
+import duke.task.ToDo;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Perhaps you want to put line spacing in between the imports from different packages to make it more readable? I notice the same in some other files.

```suggestion
package duke;

import duke.Exception.DukeException;
import duke.task.ToDo;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;
```